qfieldsync/ui/package_dialog.ui

```diff
@@ -238,9 +254,22 @@
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
-       <widget class="QLabel" name="infoLabel">
+       <widget class="QLabel" name="infoConfigurationLabel">
         <property name="text">
-         <string>Some layers in this project have not yet been configured. &lt;a href=&quot;configuration&quot;&gt;Configure project now&lt;/a&gt;.</string>
+         <string>Some layers in this project have not yet been configured, configure those now.</string>
+        </property>
+        <property name="wordWrap">
+         <bool>true</bool>
+        </property>
+       </widget>
+      </item>
+      <item row="1" column="0">
+       <widget class="QLabel" name="infoLocalizedPresentLabel">
+        <property name="text">
+         <string>Some layers are stored in localized data paths, make sure those are copied into the localized data path of devices running QField. On most devices, the path is /QField/basemaps.</string>
```

Comment:
```suggestion
         <string>Some layers are stored in localized data paths, make sure those are copied into the localized data path of devices running QField. On most devices, the path is <pre>/QField/basemaps.</pre></string>
```

That will make it clear this belongs together (compare the break placed in a strange position in your screenshots)

Reply:
What would be very nice (easy to do?): print a list of layer /filenames.